src/Namshi/Notificator/Notification/Email/EmailNotification.php

```diff
@@ -9,33 +9,30 @@
  */
 class EmailNotification extends Notification implements EmailNotificationInterface
 {
-    protected $recipientAddress;
-    
     /**
-     * Constructor.
-     * 
-     * @param string $recipientAddress
-     * @param array $parameters
+     * Recipient Email Addresses.
+     *
+     * @var array
      */
-    public function __construct($recipientAddress, array $parameters = array())
-    {
-        $this->setRecipientAddress($recipientAddress);
-        $this->setParameters($parameters);
-    }
+    protected $recipientAddresses = [];
 
     /**
-     * @inheritDoc
+     * Constructor.
+     * 
+     * @param array|string $recipientAddress
+     * @param array $parameters
      */
-    public function getRecipientAddress()
+    public function __construct($recipientAddresses, array $parameters = array())
     {
-        return $this->recipientAddress;
+        $this->recipientAddresses = is_array($recipientAddresses) ? $recipientAddresses : [$recipientAddresses];
```

Comment:
btw [this](https://github.com/symfony/Finder/blob/master/Finder.php#L667-L684) is how symfony does it